src/lib/utils/padScale.js

```diff
@@ -41,7 +57,7 @@ export default function padScale(scale, padding) {
 	const [d1, d2] = scale.domain().map(d => {
 		return isTime ? lift(d.getTime()) : lift(d);
 	});
-	const [r1, r2] = scale.range();
+	const [r1, r2] = /** @type {any[]} */ (scale.range());
```

Comment:
Should this be `number[]` instead of `any[]`?

Reply:
Yes, `[number, number]` even.